src/features/DocumentFormatter.ts

```diff
@@ -384,6 +467,11 @@ export class DocumentFormatterFeature implements IFeature {
         this.rangeFormattingEditProvider = vscode.languages.registerDocumentRangeFormattingEditProvider(
             "powershell",
             this.documentFormattingEditProvider);
+        this.onTypeFormattingEditProvider = vscode.languages.registerOnTypeFormattingEditProvider(
+            "powershell",
+            this.documentFormattingEditProvider,
+            this.firstTriggerCharacter,
+            ...this.moreTriggerCharacters);
```

Comment:
Yes, they could just have one argument taking in triggerCharacters instead of this!